### ISSUE TITLE
ONNX import: SimplifiedLayerNormalization lowers to LayerNorm, not RMS norm

### ISSUE THREAD

**Issue by czoli1976** [sonos/tract#2646]
`SimplifiedLayerNormalization` currently lowers to `LayerNorm`, which subtracts the mean. The operator is RMS normalization: it scales by the root mean square and does **not** centre the input. Models importing cleanly and running to completion therefore produce wrong activations, with no error anywhere.

Found while bringing an ORT-GenAI fused export (Qwen3 architecture) through tract: tract's logits diverged from onnxruntime, and bisecting the intermediates put the first divergence on this node.

## Repro

A single node, with an input whose mean is deliberately non-zero so the two formulas separate:

```python
import numpy as np, onnx, onnxruntime
from onnx import helper, TensorProto
vi = helper.make_tensor_value_info
node = helper.make_node("SimplifiedLayerNormalization", ["X", "W"], ["Y"], axis=-1, epsilon=1e-5)
g = helper.make_graph([node], "sln",
                      [vi("X", TensorProto.FLOAT, [3, 8]), vi("W", TensorProto.FLOAT, [8])],
                      [vi("Y", TensorProto.FLOAT, [3, 8])])
m = helper.make_model(g, opset_imports=[helper.make_opsetid("", 17),
                                        helper.make_opsetid("com.microsoft", 1)])
m.ir_version = 10
onnx.save(m, "sln.onnx")

r = np.random.RandomState(5)
d = dict(X=(r.randn(3, 8) * 2 + 1).astype(np.float32), W=(r.rand(8) + 0.5).astype(np.float32))
np.savez("sln_in.npz", **d)
y = onnxruntime.InferenceSession("sln.onnx").run(None, d)[0]

x, w = d["X"].astype(np.float64), d["W"].astype(np.float64)
rms = x / np.sqrt((x ** 2).mean(-1, keepdims=True) + 1e-5) * w
ln = (x - x.mean(-1, keepdims=True)) / np.sqrt(x.var(-1, keepdims=True) + 1e-5) * w
print("ORT vs RMSNorm  ", np.abs(y - rms).max())   # 2.6e-07
print("ORT vs LayerNorm", np.abs(y - ln).max())    # 1.8e-01
```

```
tract sln.onnx --input-facts-from-bundle sln_in.npz run \
  --input-from-bundle sln_in.npz --assert-output-bundle sln_in.npz
```

On `d5d2fb904` tract's output matches the **LayerNorm** expression to 1.5e-07 and differs from onnxruntime by 1.8e-01. First few values of one row:

```
onnxruntime  [-2.1760528  -1.8019617  -0.531172    0.21287762]
tract        [-2.0468385  -1.6825728  -0.4022511  0.375858  ]
```

## Where it comes from

Before #2409 the op was registered on `rms_norm::rms_normalization`, which is correct. #2409 (titled "SimplifiedLayerNormalization mapped to RMSNorm instead of LayerNorm") moved it onto a new `simplified_layer_norm` handler that constructs `LayerNorm`:

https://github.com/sonos/tract/blob/main/onnx/src/ops/nn/simplified_layer_norm.rs

Note `SkipSimplifiedLayerNormalization` right below it in the registry is unaffected — it goes to `rms_norm_contrib` and is correct — so a model that fuses most of its norms into the Skip form shows the discrepancy only where the plain op survives, which makes it easy to miss.

## How to fix

Point the registration back at `rms_normalization` and delete the handler; `LayerNorm::new` then has no callers and can go too. `LayerNormalization` keeps using `layer_norm::layer_norm` and is unaffected.

```diff
--- a/onnx/src/ops/nn/mod.rs
+++ b/onnx/src/ops/nn/mod.rs
@@ -27,7 +27,6 @@ mod reduce;
 mod rms_norm;
 mod rms_norm_contrib;
 mod rotary_embedding;
-mod simplified_layer_norm;
 mod skip_layer_norm;
 
@@ -101,7 +100,7 @@ pub fn register_all_ops(reg: &mut OnnxOpRegister) {
     reg.insert("RMSNormalization", rms_norm::rms_normalization);
     reg.insert("RotaryEmbedding", rotary_embedding::rotary_embedding);
-    reg.insert("SimplifiedLayerNormalization", simplified_layer_norm::simplified_layer_norm);
+    reg.insert("SimplifiedLayerNormalization", rms_norm::rms_normalization);
```

plus `rm onnx/src/ops/nn/simplified_layer_norm.rs` and dropping the now-unused `impl LayerNorm { pub fn new(..) }` block in `onnx/src/ops/nn/layer_norm.rs`.

With that applied, the single-node repro above matches onnxruntime to 7.5e-09.

One wrinkle if this gets a case under `onnx/test_cases/`: tract's and onnxruntime's `rsqrt` disagree in the last f32 ulp, so `--assert-output-bundle` at the default `close` tolerance reports one outlier in 24. `run_all.sh` passes `$OPTIONS` before the `run` subcommand, so `--approx` cannot be threaded through from `vars.sh` as it stands.
